src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceRadio.vue

```diff
@@ -1,15 +1,35 @@
 <template>
-  <v-radio-group>
-    <v-radio label="Radio One" value="one"></v-radio>
-    <v-radio label="Radio Two" value="two"></v-radio>
-    <v-radio label="Radio Three" value="three"></v-radio>
+  <div>
+    <h3>
+    {{ props.title }}
+  </h3>
+  <br />
+  <v-radio-group :rules="[Rules.requiredRadio()]" @update:model-value="(value) => $emit('update:model-value', value as string)">
```

Comment:
I think the :rules should be set by props.rules. Like in the text-input component 

https://github.com/bcgov/ECC-ECER/blob/master/src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceTextField.vue